packages/v4/patternfly-docs/content/design-guidelines/components/password-strength/password-strength.md

```diff
@@ -1,6 +1,6 @@
 ---
 id: Password strength
-section: demos
+section: components
```

Comment:
Just confirming that this is the only item in this PR moving to 'components' (though it is not a standalone component)?

Reply:
It's not the only item moving, but it's the only item moving that has associated design guidelines. Therefore it's the only existing demo that requires changes in Org. Does that make sense?

Reply:
I think - just confirming "components" is the right section for this to live